src/libraries/System.Private.CoreLib/src/System/Runtime/CompilerServices/ConditionalWeakTable.cs

```diff
@@ -48,8 +48,10 @@ public ConditionalWeakTable()
         /// </param>
         /// <returns>Returns "true" if key was found, "false" otherwise.</returns>
         /// <remarks>
-        /// The key may get garbaged collected during the TryGetValue operation. If so, TryGetValue
+        /// The key may get garbage collected during the TryGetValue operation. If so, TryGetValue
         /// may at its discretion, return "false" and set "value" to the default (as if the key was not present.)
+        /// This method needs to follow the rules in RestrictedCallouts.h, as it may be called
```

Comment:
This should not be doc comment. It should be regular comment instead. It would not make sense to reference RestrictedCallouts.h in the public docs (https://learn.microsoft.com/dotnet/api/system.runtime.compilerservices.conditionalweaktable-2.trygetvalue).

Reply:
I removed these comments about following the rules. Instead I added a test.